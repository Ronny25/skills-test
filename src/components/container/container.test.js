import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import Container from './container';
import LoadingSpinner from '../loading-spinner';

describe('<Container />', function () {

  it('should renders correctly', () => {
    const tree = renderer
      .create(<Container />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render LoadingSpinner', () => {
    const wrapper = mount(
      <Container isLoading />
    );

    wrapper.setState({ isLoading: true });

    expect(wrapper.find(LoadingSpinner)).toHaveLength(1);
  });

});
