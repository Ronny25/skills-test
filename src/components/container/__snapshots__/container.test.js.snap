// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should renders correctly 1`] = `
<div>
  <div
    className="css-otbafj"
  >
    <div
      className="css-1omkubd"
    >
      <input
        className="css-1wwkbse"
        onChange={[Function]}
        placeholder="What are you looking for?"
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="css-ihdrx8"
  />
</div>
`;
